rpc/types.go

```diff
@@ -302,21 +302,19 @@ type BroadcastedInvokeTxnV1 struct {
 	InvokeV1
 }
 
-type ContractEntryPoint struct {
+type EntryPoint struct {
 	// The offset of the entry point in the program
 	Offset NumAsHex `json:"offset"`
 	// A unique identifier of the entry point (function) in the program
 	Selector string `json:"selector"`
 }
 
-type ContractEntryPointList []ContractEntryPoint
-
-type ContractABI []ContractABIEntry
+type ABI []ABIEntry
 
 type EntryPointsByType struct {
-	CONSTRUCTOR ContractEntryPointList `json:"CONSTRUCTOR"`
-	EXTERNAL    ContractEntryPointList `json:"EXTERNAL"`
-	L1_HANDLER  ContractEntryPointList `json:"L1_HANDLER"`
+	Constructor []EntryPoint `json:"CONSTRUCTOR"`
```

Comment:
prefer to use an plain array type over a type alias. the type alias is harder to grok, since you need to read the full name (which doesnt have an enforces scheme) or follow the declaration to see what is aliased